rotkehlchen/api/v1/encoding.py

```diff
@@ -2468,6 +2468,7 @@ class Eth2ValidatorSchema(Schema):
         ),
     )
     public_key = fields.String(load_default=None)
+    ownership_percentage = FloatingPercentageField(load_default=FVal(1))
```

Comment:
I  guess you tested this, but load_default skips the deserialization?

Reply:
Yes, I tested it here https://github.com/rotki/rotki/blob/e1eba5f80abda901160c8ffe9a38c336dcf9f32a/rotkehlchen/tests/api/test_eth2.py#L258

the tests makes the addition of the validators (with and without the `ownership_percentage`) and then compares to the serialization that contains the `ownership_proportion`